js/main.js

```diff
@@ -88,21 +93,21 @@ const pinsCreate = (object) => {
   return pinTemplate;
 };
 
-const pinsFragment = (pinsData) => {
+const getPinsFragment = (pinsData) => {
   const fragment = document.createDocumentFragment();
 
   for (let i = 0; i < pinsData.length; i++) {
-    fragment.appendChild(pinsCreate(pinsData[i]));
+    fragment.appendChild(pinCreate(pinsData[i]));
   }
 
   return fragment;
 };
 
-const pinsScreen = () => {
-  const pinsDataArr = createAds(PINS_COUNT);
-  const pinsFragmentNodes = pinsFragment(pinsDataArr);
+const getPinsScreen = () => {
```

Comment:
 префикс `get` у названия функции обозначает, что функция нам что-то возвращает, тут же возврата нет, потому уместнее:
```suggestion
const renderPinsScreen = () => {
```

-----
_Примечание_ По образу действия функции, которые что-то возвращают называют **функция**, а если они ничего не возвращают, но вносят изменения в документ/файл/переменные, то их называют **процедура**, это должно отражаться в именовании согласно англ языку